packages/webapp/src/components/Table/v2/index.jsx

```diff
@@ -0,0 +1,282 @@
+    setOrderBy(property);
+  };
+
+  const handleRowClick = (event, index) => {
+    if (onRowClick) {
+      onRowClick(event, index);
+    }
+  };
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  const onClickLoadMore = () => {
+    const rowCount = Math.min(rowsPerPage + itemsToAddPerLoadMoreClick, data.length);
+    setRowsPerPage(rowCount);
+  };
+
+  // Avoid a layout jump when reaching the last page with empty rows.
+  const emptyRows = page > 0 ? Math.max(0, (1 + page) * rowsPerPage - data.length) : 0;
+
+  const visibleRows = useMemo(
+    () =>
+      data
+        .slice()
+        .sort(getComparator(order, orderBy))
+        .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage),
+    [order, orderBy, page, rowsPerPage],
+  );
+
+  return (
+    <Box sx={{ width: '100%' }}>
+      <TableContainer>
+        <Table
+          aria-labelledby="tableTitle"
+          className={clsx(styles.table, shouldFixTableLayout && styles.fixed)}
+        >
+          <EnhancedTableHead
+            columns={columns}
+            order={order}
+            orderBy={orderBy}
+            onRequestSort={handleRequestSort}
+            dense={dense}
+          />
+          <TableBody>
+            {visibleRows.map((row, index) => {
+              return (
+                <TableRow
+                  key={index}
+                  onClick={(event) => handleRowClick(event, row)}
+                  className={clsx(styles.tableRow, onRowClick && styles.clickable)}
+                >
+                  {columns.map(({ id, format, accessor, align, columnProps }) => {
+                    if (!id) {
+                      return null;
+                    }
+
+                    const dataGetter = accessor || id;
+                    const data =
+                      typeof dataGetter === 'function' ? dataGetter(row) : row[dataGetter];
+
+                    return (
+                      <TableCell
+                        key={id}
+                        className={clsx(styles.tableCell, dense && styles.dense)}
+                        align={align || 'left'}
+                        {...columnProps}
+                      >
+                        {format ? format(row) : data}
+                      </TableCell>
+                    );
+                  })}
+                </TableRow>
+              );
+            })}
+            {emptyRows > 0 && (
+              <TableRow style={{ height: (dense ? 40 : 56) * emptyRows }}>
+                <TableCell colSpan={fullColSpan} className={styles.tableCell} />
+              </TableRow>
+            )}
+            {!showPagination && (
+              <More
+                onClickMore={onClickMore}
+                onClickLoadMore={onClickLoadMore}
+                invisibleRowCount={!showPagination && data.length - rowsPerPage}
```

Comment:
Since we're already checking for `!showPagination` above this could be
```suggestion
                invisibleRowCount={data.length - rowsPerPage}
```